std/json.d

```diff
@@ -628,7 +655,7 @@ struct JSONValue
     /// Implicitly calls $(D toJSON) on this JSONValue.
     ///
     /// $(I options) can be used to tweak the conversion behavior.
-    string toString(in JSONOptions options = JSONOptions.none) const
+    string toString(in JSONOptions options = JSONOptions.none) const @safe
     {
         return toJSON(&this, false, options);
```

Comment:
`&this` should probably not be allowed in `@safe` for the same reason `&refParam` isn't, but we'll be reminded of that when we fix it in the compiler anyway